### DISCUSSION TITLE
ft.InputFilter: change behaviour

### DISCUSSION THREAD

**Discussion by ClearSafety** [flet-dev/flet#3764] [Ideas]
The class ft.InputFilter makes the field content be erased if a not allowed charactered is typed by the user.
On the other hand, the class ft.NumbersOnlyInputFilter() does not do it, it simply does not allow the not allowed typed value been entered in the field.

Using ft.NumbersOnlyInputFilter(), if you type 'd', for instance, nothing happens.
Using ft.InputFilter(regex_string='^\d*(\.\d{0,2})?$'), if you type 'd', everything previously type is deleted.

I'd like to suggest that 'InputFilter' could have the same behaviour of NumbersOnlyInputFilter

**Comment by ndonkoHenri** [comment]
Created an issue: https://github.com/flet-dev/flet/issues/3769
